src/app/landing/LandingPage.css

```diff
@@ -249,3 +254,26 @@
     font-size: 11px;
   }
 }
+
+@media screen and (max-width: 1224px) {
+  .LandingPage .IntroBody {
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6)),url('/img/landing_page/top1_mobile.jpg') no-repeat center center fixed;
+    background-position: 30% 0%;
```

Comment:
Doesn't hurt to have background-position and background-size here, but do they actually make any difference? As far as I can tell they would have these values anyway, thanks to the existing `.LandingPage .IntroBody` styles above, outside the breakpoint.

Reply:
There was a weird issue so I left it in but it's fixed now so I'm removing the redundant code